### PR TITLE
Enterprise 1.9.8  release notes

### PR DESCRIPTION
**Pull request by lwandzura**
Closes https://github.com/influxdata/docs-v2/issues/4124

1.9.8 release notes

@samhld after approving, please let us know what date we can publish

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md [hunk 1]
```diff
@@ -9,13 +9,8 @@ menu:
...
-
-{{% warn %}}
-An edge case regression was introduced into this version that may cause a constant build-up of hinted handoff if writes are rejected due to malformed requests. We're reverting back to InfluxDB Enterprise 1.9.6 as the official stable version. If you experience write errors and hinted hand-off growth, we recommend reverting back to 1.9.6 or upgrading to 1.9.8 when released.
-{{% /warn %}}
+## 1.9.8 [2022-07-05]
 
-<!--
 ### Features
 - Expose passive node feature to influxd-ctl and the API.
 - Throttle inter-node data replication, both incoming writes and hinted hand-off, when errors are encountered.
```
Comment (samhld) [inline content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:16]:
```suggestion
- Throttle inter-node data replication, both incoming writes and hinted hand-off, when errors are encountered.
- Add new `hh_node` measurement to the `/debug/vars` monitoring telemetry. This tracks more accurately with the `max-size` configuration setting for hinted handoff in data nodes.
```

**Review by samhld — CHANGES_REQUESTED** [review]

File: content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md (hunk 1 above)
Comment (lwandzura) [reply content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md:16]: Not sure how you are adding updates but a lot of these fixes you are suggesting don't actually need to be made, they look correct in VS Code and render without issue in our mock up server. I did add the hh_node part which was not included in this release notes issue https://github.com/influxdata/docs-v2/issues/4124.

File: content/enterprise_influxdb/v1.9/concepts/glossary.md [hunk 2]
```diff
@@ -245,11 +243,9 @@ Describes how long InfluxDB keeps data (duration), how many copies of the data t
 When you create a database, InfluxDB creates a retention policy called `autogen` with an infinite duration, a replication factor set to one, and a shard group duration set to seven days.
 For more information, see [Retention policy management](/enterprise_influxdb/v1.9/query_language/manage-database/#retention-policy-management).
 
-Related entries: [duration](#duration), [measurement](#measurement), [replication factor](#replication-factor), [series](#series), [shard duration](#shard-duration), [tag set](#tag-set)
+Related entries: [duration](#duration), [measurement](#measurement), [replication factor](#replication-factor), [series](#series), [shard duration](#shard-duration), [tag set](#tag-set)\
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/concepts/glossary.md:246]:
```suggestion
Related entries: [duration](#duration), [measurement](#measurement), [replication factor](#replication-factor), [series](#series), [shard duration](#shard-duration), [tag set](#tag-set)
```

**Review by kelseiv — APPROVED** [review]
we'll need to update the release date as needed

**Pull request merged**